src/components/SettlementButton.tsx

```diff
@@ -144,7 +144,8 @@ function SettlementButton({
     const {isOffline} = useNetwork();
 
     const session = useSession();
-    const chatReport = ReportUtils.getReport(chatReportID);
+    // The chatReportID can be an empty string that can make the app crashes if we use a nullish ?? operator
```

Comment:
The reason the app would crash is due to subscribing to the entire report collection in the case of an empty string, right? If so, I would mention that explicitly in the comment so it's not left to wonder why it would cause the app to crash.

Reply:
> The reason the app would crash is due to subscribing to the entire report collection in the case of an empty string, right

Yes

Reply:
OK, please indicate that in your code comment.

Reply:
I updated.